src/Sylius/Bundle/AttributeBundle/Attribute/AsAttributeType.php

```diff
@@ -22,6 +22,7 @@ public function __construct(
         private string $type,
         private string $label,
         private string $formType,
+        private ?string $configurationFormType = null,
         private int $priority = 0,
```

Comment:
```suggestion
        private int $priority = 0,
        private ?string $configurationFormType = null,
```
to be backward compatible